articles/azure-maps/power-bi-visual-get-started.md

```diff
@@ -31,34 +31,31 @@ The Azure Maps Power BI visual connects to cloud service hosted in Azure to retr
 - Data in the Location, Latitude, and Longitude buckets may be sent to Azure to retrieve map coordinates (a process called geocoding).
 - Telemetry data may be collected on the health of the visual (for example, crash reports), if the telemetry option in Power BI is enabled.
 
-Other than the scenarios described above, no other data overlaid on the map is sent to the Azure Maps servers. All rendering of data happens locally within the client.
+Other than the scenarios previously described, no other data overlaid on the map is sent to the Azure Maps servers. All rendering of data happens locally within the client.
 
-You, or your administrator, may need to update your firewall to allow access to the Azure Maps platform that uses the following URL.
+> [!TIP]
+> If using the Azure Maps [Geographic API endpoints], your firewall may need to be updated to allow access to the Azure Maps platform using either or both of the following URLs:
+>
```

Comment:
We should still keep `https://atlas.microsoft.com` since the map control, and geocoding requests for tenants outside EU /US will still use this URL.